aspnetcore/signalr/hubcontext.md

```diff
@@ -0,0 +1,60 @@
+---
+title: SignalR HubContext
+author: rachelappel
+description: Overview of the use of SignalR's HubContext
+manager: wpickett
+monikerRange: '>= aspnetcore-2.1'
+ms.author: rachelap
+ms.custom: mvc
+ms.date: 6/12/18
+ms.prod: aspnet-core
+ms.technology: aspnet
+ms.topic: article
+uid: signalr/HubContext
+---
+
+# Sending Messages From Outside a Hub
+
+By [Mikael Mengistu](https://github.com/mikaelm12)
+
+The SignalR Hub is the core abstraction for sending messages to clients connected to our SignalR server. However, it is also possible to send messages from other places in your application using the IHubContext service. Here we'll look at how to access a SignalR IHubContext to send notification to clients from outside of a hub.
+
+[View or download sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/signalr/hubcontext/sample/) [(how to download)](xref:tutorials/index#how-to-download-a-sample)
+
+## Get an instance of `HubContext`
+
+In ASP.NET Core SignalR you can get access to an instance of IHubContext via dependency injection. You can inject an instance of IHubContext<YourHubType> into a Controller, Middleware or other DI service and use it to send messages to clients..
```

Comment:
nit: got a double-`.` at the end (a "Fuller Stop" one might say?)

Reply:
Also your `<>` are getting lost because they aren't in `code ticks` so they're being treated like HTML tags. Just put all references to `IHubContext` in those ticks ;)